### PR TITLE
uploading assignment 5

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

File: poonam/html/assignment5.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<html>
+    <head>
+        <title>Assignment 5</title>
+        <link rel="Stylesheet" href="assignment5.css">
+    </head>
+    <body style="border: 1px solid black;">
+        <div style="background: white; border: 1px solid black;">
+            <h1>Poonam Khode</h1>
+        </div>
+        <div class="div">
```
Comment (VIP5614) [inline poonam/html/assignment5.html]: div na rakh class name box kar dijiye ok

File: poonam/html/assignment5.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            <h3 class="h3">About Me</h3>
+            <div class="div2">
```
Comment (VIP5614) [inline poonam/html/assignment5.html]: same problem

File: poonam/html/assignment5.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                <P>
+                    Hello!!<br>
+                    My name is Poonam Khode.<br>
+                    I am from Nagpur.<br>
+                    I have completed my graduation in Bsc-IT<br>
+                    From SRWC, Nagpur which is affialated to SNDT University.<br>
+                    My family consists of 3 members including me, my mother and grand mother.<br>
+                </P>
+            </div>
+            <h3 class="h3 m">More About Me</h3>
```
Comment (VIP5614) [inline poonam/html/assignment5.html]: same problem

File: poonam/html/assignment5.html (hunk 1 above)
Comment (poonam-khode) [reply poonam/html/assignment5.html]: okay

File: poonam/html/assignment5.html (hunk 3 above)
Comment (pratikshaghule) [reply poonam/html/assignment5.html]: Dont use the tag as class or id name

**Review by VIP5614 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**